service/README.md

```diff
@@ -48,6 +48,21 @@ You should get a response similar to this:
   {"status":"UP","diskSpace":{"status":"UP","total":249644974080,"free":137188298752,"threshold":10485760}}
 ```
 
+Ensure variable for CCD_DEF_CASE_SERVICE_BASE_URL in bin/import-ccd-definition.sh 
+is set to http://fpl-service:4000 - this ensures the CCD use the docker networking to reach 
+the service.  
+
+### Run application from IntelliiJ
+For local profile - configure the notify.api_key if necessary (in service/src/main/resources/application.yaml). 
```

Comment:
Please add blank line between headline and paragraph. Despite the fact it does not matter final render I tend to read raw Markdown files - it will make my life easier.

I know that we have no consistency there :(

Reply:
Sure! Done.